docs/content/modules/tensor_mechanics/systems/Kernels/StressDivergence1DRSpherical.md

```diff
@@ -1,4 +1,4 @@
-<!-- MOOSE System Documentation Stub: Remove this when content is added. -->
-#StressDivergence1DRSpherical System
-!parameters /Kernels/StressDivergence1DRSpherical
+#Stress Divergence 1D RSpherical Action
```

Comment:
Breaking up the file name with spaces helps to not overrun the blue left sidebar block